remote_test.go

```diff
@@ -1708,6 +1743,21 @@ var framePage = `
 </html>
 `
 
+var titleChangePage = `
+<html>
+<head>
+	<title>Go Selenium Test Suite - Title Change Page</title>
+</head>
+<body>
+	This page will change a title after 5 seconds.
+
+	<script>
+		setTimeout(function() { document.title = 'Title changed.' }, 5000);
```

Comment:
Given that the existing tests run so long already (and I'm about to start work to make them faster, see #79), I would like to avoid having this test take a minimum of ten seconds.

I would structure the test using goroutines and use channels as barriers. This way, we can control the timing of the events much more closely, and make the test faster.

(The following is untested.)

```go
if err := wd.Get(serverURL); err != nil {
...
}

const newTitle = "Title changed"
ch := make(chan bool)
titleChangeCondition := (wd WebDriver) (bool, error) {
  ... body same as earlier, using the newTitle constant ...

  ch <- true
}

go func() {
  if err := wd.Wait(titleChangeCondition); err != nil {
    t.Fatalf("wd.Wait(titleChangeCondition) returned error: %v", err)
  }
  close(ch)
}()

<-ch  // wait for the condition to run once.
script := fmt.Sprintf(`document.title = %q', newTitle)
if _, err := wd.ExecuteScriptRaw(script, nil); err != nil {
...
}

<-ch // wait for the condition to become true
```